matlab/src/matlab/+arrow/+type/ID.m

```diff
@@ -41,7 +41,7 @@
         % IntervalDayTime (22)
         % Decimal128      (23)
         % Decimal256      (24)
-        % List            (25)
+        List      (25)
```

Comment:
Could you align `(25)` and `(26)`?

Reply:
Sure. I think it makes sense to just align all of the values to the longest type name.

Reply:
Fixed.